### ISSUE TITLE
How to use keyboard navigation? 

### ISSUE THREAD

**Issue by hakonhagland** [valentine195/obsidian-settings-search#15]
Thanks for creating this nice plugin.

The [README](https://github.com/jmilldotdev/obsidian-frontmatter-tag-suggest/blob/master/README.md) file says:

> You can use the keyboard to navigate the search results. The up and down arrow keys can be used to move through the results, and the enter button will take you to the setting.

If I press `CTRL+,` to display the settings dialog window and then type "core pl" into the search field in the top left corner (let's say I want to browse core plugin settings) the top item in "Settings search results" in the right pane is "Search core plugins" but how can I use keyboard navigation to activate it? Tab does not work, arrow keys does not work, enter does not work. Also, I was used to that pressing ESC would close the settings dialog. However, now ESC does nothing and I have to manually close the dialog by pressing the cross mark in the top right corner. If I do this, and close the dialog and then reopen it again by pressing CTRL+, the search field is already filled out with "core pl" but the   "Settings search results" in the right pane does not show "Search core plugins" any more as the top result.

I am on Linux using Obsidian version 0.14.2

**Comment by valentine195** [comment]
I do not believe this is an issue anymore....
